packages/block-editor/src/store/private-selectors.js

```diff
@@ -547,6 +547,13 @@ export function isZoomOutMode( state ) {
 	return state.editorMode === 'zoom-out';
 }
 
+/**
+ * Retrieves the root client ID for a section from the state.
```

Comment:
```suggestion
 * Retrieves the client ID of the block which contains the blocks
 * acting as "sections" in the editor. This is typically the "main content"
 * of the template/post.
```

It's a tricky concept to explain. Essentially there is an algorithm which finds the block which contains the "sections" on the page.

Reply:
I think you did a good job here, @getdave :)